experimental/python/databricks/__init__.py

```diff
@@ -0,0 +1,3 @@
+# DO NOT ADD ANYTHING ELSE TO THIS FILE FOR COMPATIBILITY WITH OTHER databricks.* PACKAGES # noqa
+# SEE https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # noqa
```

Comment:
According to https://github.com/databricks/databricks-sdk-py/blob/main/databricks/__init__.py#L4

> The contents of this file must be byte-for-byte equivalent across all packages.

Reply:
Fixed, it turns out this file was updated in the mean time